WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndValueRow.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+struct TitleAndValueRow: View {
```

Comment:
Do you think some comments about how and when to use these new views would be useful?